test/e2e/mxGraph.model.test.ts

```diff
@@ -211,6 +220,29 @@ describe('mxGraph model', () => {
 `;
   const bpmnVisu = new BpmnVisu(null);
 
+  function expectFont(state: mxgraph.mxCellState, expectedFont: ExpectedFont): void {
```

Comment:
:+1: cool to have that kind of tests
I guess this was required because fonts was defined in the input bpmn source :smile:

Reply:
:question: should we introduce another label with another font in bold (for instance) to be used by a task? this would ensure that other argument are parsed correctly.
Otherwise we could add a unit test on the computeStyle method to increase test coverage

Reply:
Done